www/js/search.js

```diff
@@ -31,15 +31,15 @@ const searchInputs = h('div#search-container', [
     onfocus: e => module.exports.focusSearch(e),
     onkeyup: e => module.exports.typeSearch(e),
   }),
-  h('div#search-bg'),
-  h(
-    'button#search-options-toggle',
-    {
-      type: 'button',
-      onclick: e => module.exports.toggleSearchOptions(e),
-    },
-    h('i.fa.fa-cog'),
-  ),
+  h('span', 'Ang'),
+  h('input#ang-input.gurmukhi', {
+    type: 'number',
```

Comment:
Clicking on the arrows doesn't update the search results. I think the event fired is different. maybe `oninput` ? could also hide them with css. 
![numberpicker](https://user-images.githubusercontent.com/2982784/43358437-96bd8cf2-9246-11e8-8056-cf9a15533a0d.gif)